spot_tools_ros/src/spot_tools_ros/spot_executor_ros.py

```diff
@@ -254,13 +255,37 @@ def tf_lookup_fn(parent, child):
             10,
         )
 
+        if not use_fake_spot_interface:
+            self.pose_pub = self.create_publisher(Pose2D, "~/spot_pose", 1)
+            timer_period = 1.0  # seconds
+            self.pose_timer = self.create_timer(timer_period, self.publish_pose)
+
         self.heartbeat_pub = self.create_publisher(NodeInfoMsg, "~/node_status", 1)
         heartbeat_timer_group = MutuallyExclusiveCallbackGroup()
         timer_period_s = 0.1
         self.timer = self.create_timer(
             timer_period_s, self.hb_callback, callback_group=heartbeat_timer_group
         )
 
+    def publish_pose(self):
```

Comment:
Can we extend this to be responsible for publishing the fake spot TF as well? I think it's too confusing if we publish the fake spot TF from inside FakeSpotRos but the real TF from here. Another option would be to add a SpotRos that is analogous to FakeSpotRos, which would nicely keep the non-executor functionality out of this file, but maybe that overhead isn't worth it since I don't think anything else would go into SpotRos.